src/features/announcements/index.ts

```diff
@@ -0,0 +1,29 @@
+import chicken from "assets/animals/chickens/walking.gif";
+interface Announcement {
+  date: Date;
+  image?: string;
+  title: string;
+  description: string;
+  link?: string;
+}
+
+export const announcements: Announcement[] = [
```

Comment:
For consistency I would suggest calling this variable `ANNOUNCEMENTS` with uppercase. This is currently the repo's standard for constants/config like values